pkg/controller/installation/core_controller_test.go

```diff
@@ -1002,7 +1002,7 @@ var _ = Describe("Testing core-controller installation", func() {
 			})
 		})
 
-		Context("with LinuxDataplane=BPF and BPFNetworkBootstrap=Enabled", func() {
+		Context("streamline BPF installation", func() {
```

Comment:
I actually think the previous description of this context was more clear / accurate!